docs/tern-lab-tutorial.md

```diff
@@ -0,0 +1,135 @@
+# Tern
+
+Tern is an inspection tool to find the metadata of the packages installed in a container image. Tern gives you a deeper understanding of your container's bill of materials so you can make better decisions about your container based infrastructure, integration and deployment strategies. It's also a good tool if you are curious about the contents of the container images you have built. This lab will show you how you can use Tern to inspect a Photon container image and also inspect the image from a resulting Dockerfile.
+
+## Environment Overview
+
+This lab tutorial assumes that you are working in a Linux environment. Your lab environment is hosted with a cloud server instance. This instance is accessible by clicking the "My Lab" icon on the left of your Strigo web page. You can access the terminal session using the built-in interface provided by Strigo or feel free to SSH directly as well (instructions will appear in the terminal).
+
```

Comment:
This is OK to leave here. I think it's reasonable to say that this tutorial was built as a Strigo tutorial but can be followed if you are working in a Linux environment.

Reply:
@nishakm Sounds good, I will make the changes.